pypi_README.md

```diff
@@ -0,0 +1,16 @@
+zcbor
+=====
+
+After installing, zcbor can be invoked on the command line as `zcbor`, or included as a module via `import zcbor`.
+
+This package has 2 uses:
+
+1. Schema-based validation and conversion of YAML/JSON/CBOR data.
+2. Generating C code that decodes/encodes and validates CBOR data according to a schema.
+
+The schema language used by zcbor is [CDDL](https://datatracker.ietf.org/doc/rfc8610/) which allows creating very advanced and detailed schemas.
+
+The PyPi package comes with a C library which is needed by the generated code.
```

Comment:
Out of curiosity, how do you use the C lib from the PyPi package?

Reply:
Good point. It's there so it can be used by the user's project along with generated code, it's not run directly by the package. I will clarify the text.